### DISCUSSION TITLE
How can l add my wallet  ( Cüzdan kaydetme )

### DISCUSSION THREAD

**Discussion by ahmederoni** [IBAX-io/go-ibax#746] [Q&A]
l opened a wallet but l can not find where to save the wallet 

Ben cüzdan açtım ama cüzdanı nereye kaydedecegimi bulamıyorum,reedem seçenegi vardı ama site degiştigi için bulamıyorum

**Comment by Shekilsheyy** [comment]
Bulman lazım hocam ekran görüntüsü almadınız mı

**Reply by ahmederoni** [reply]
cüzdan için private key vs kaydettim orda sorun yok,ben cüzdanı kaydedecegim bölümü bulamıyorum

**Reply by herteldennnn** [reply]
bende bulamıyorum

**Comment by MertSade** [comment]
https://ibax.io/user/redeem buradan ulaşabilirsin.
